### ISSUE TITLE
🐛 Some packages' sub files are unable to be accessed

### ISSUE THREAD

**Issue by prateekbh** [CompuIves/codesandbox-client#797]
As stated in https://twitter.com/codesandboxapp/status/992483491269890049.

I am unable to access the files inside this package.

**Comment by prateekbh** [comment]
any updates?

**Comment by CompuIves** [comment]
Ah sorry for the late response. I've tried several services to get access to the file, but I haven't found a service that can handle the size of the dependency yet. We normally do this call: https://unpkg.com/preact-material-components@1.5.0/?meta to get the list of files/dirs. The last time I tried that endpoint we got 10MB of data about files alone, and now it seems to time out.

One solution that could work is getting the meta of the sub directory (https://unpkg.com/preact-material-components@1.5.0/Card/?meta). However, as soon as our resolver needs data from the root it will hang again. This issue is super interesting, I think it can be solved, but I haven't found the best way to handle it yet. I can't give a timeline on when it's fixed, it could be this week or in a month, mostly because I need to get the idea on how to fix it 😄.
